program/src/processor.rs

```diff
@@ -309,23 +301,23 @@ pub fn process_unwrap(accounts: &[AccountInfo], amount: u64) -> ProgramResult {
     let bump_seed = [bump];
     let signer_seeds = get_wrapped_mint_authority_signer_seeds(wrapped_mint.key, &bump_seed);
 
-    invoke_signed(
-        &spl_token_2022::instruction::transfer_checked(
-            unwrapped_token_program.key,
-            unwrapped_escrow.key,
-            unwrapped_mint.key,
-            recipient_unwrapped_token.key,
-            wrapped_mint_authority.key,
-            &[],
-            amount,
-            unwrapped_mint_state.base.decimals,
-        )?,
-        &[
-            unwrapped_escrow.clone(),
-            unwrapped_mint.clone(),
-            recipient_unwrapped_token.clone(),
-            wrapped_mint_authority.clone(),
-        ],
+    let mut transfer_accounts = vec![
+        unwrapped_escrow.clone(),
+        unwrapped_mint.clone(),
+        recipient_unwrapped_token.clone(),
+        wrapped_mint_authority.clone(),
+    ];
```

Comment:
Since `invoke_transfer_checked` is accepting all of these `AccountInfo`s directly, can't you just pass `additional_accounts` to `invoke_transfer_checked` as they are without creating `transfer_accounts`?

Reply:
Oh! I wonder how I never noticed that 🤔